### DISCUSSION TITLE
Unable to detect device share

### DISCUSSION THREAD

**Discussion by fimak** [Web3Auth/Web3Auth#695] [Help]
> Unable to detect device share.
> This may be due to browser security settings. Adjust your current browser settings from 'Strict' to 'Moderate'. Alternatively, login via a Chrome Browser.

I integrated web3Auth in ReactNative app using in app browser by your documentation. Everything is fine until I ask QA team to test it.
When some body else tries to test the application they get this screen.

How to avoid it?

**Comment by shahbaz17** [comment]
Hey @fimak 

Please update to the latest version of [React Native SDK](https://github.com/Web3Auth/web3auth-react-native-sdk) `v3.1.0` and reach out to us if this error is still there.

**Reply by fimak** [reply]
I've already used this version.
```
"@web3auth/react-native-sdk": "^3.1.0",
```

**Reply by fimak** [reply]
I haven't seen this error last 5 days.
But I would like to exclude potential causes of that error.

**Comment by AndonMitev** [comment]
Still exists on 0.3.1

**Reply by shahbaz17** [reply]
Can you share your log here?

Please use `chrome://inspect` to see logs of your attached application.

**Reply by shahbaz17** [reply]
Use `v3.21.1` https://github.com/Web3Auth/web3auth-react-native-sdk/releases and try to see if it is still there.
